dracut/10usr-generator/usr-generator

```diff
@@ -58,7 +59,19 @@ esac
 
 # Only proceed if the source is a path.
 if [[ "${usr}" != /* ]]; then
+  if [[ -z "${usr}" && -f /usr.squashfs ]]; then
+    # Don't set "norecovery" for squashfs
```

Comment:
i haven't come across this yet - when is /usr.squashfs present?

Reply:
This is for PXE boots